scripts/bin/update-packages

```diff
@@ -31,6 +31,9 @@ TERMUX_PACKAGES_DIRECTORIES=$(jq --raw-output 'del(.pkg_format) | keys | .[]' "$
 # shellcheck source=scripts/properties.sh
 . "${TERMUX_SCRIPTDIR}/scripts/properties.sh"
 
+# my GH_TOKEN for debugging, this is already passed in for the CI.
+. "${TERMUX_SCRIPTDIR}/gh.env" || :
```

Comment:
This is the last piece of debugging code left from testing.
Will be removed before final merge.